Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM registry.centos.org/centos/centos:7
 
 ENV LANG=en_US.UTF-8 \
-    F8A_WORKER_VERSION=d403113 \
+    F8A_WORKER_VERSION=58d3025 \
     F8A_AUTH_VERSION=5211e23 \
-    F8A_UTILS=f94a04e
+    F8A_UTILS=3bca34e
```

Comment:
Any specific reasons why utils is updated?

Reply:
I had to pull in the latest version of worker to get the RDS model. When I pulled that it required some functions that were present in a later version of utils.

Reply:
ok